smokey/src/main/java/frc/robot/commands/MoveArmToStowed.java

```diff
@@ -0,0 +1,40 @@
+// ************************************************************
+// Bishop Blanchet Robotics
+// Home of the Cybears
+// FRC - Charged Up - 2023
+// File: MoveArmToStowed.java
+// Intent: Forms a command to bring the arm to an inputed scoring position.
+// ************************************************************
+
+// ʕ •ᴥ•ʔ ʕ•ᴥ•  ʔ ʕ  •ᴥ•ʔ ʕ •`ᴥ´•ʔ ʕ° •° ʔ ʕ •ᴥ•ʔ ʕ•ᴥ•  ʔ ʕ  •ᴥ•ʔ ʕ •`ᴥ´•ʔ ʕ° •° ʔ 
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class MoveArmToStowed extends CommandBase {
```

Comment:
Naher is working on a class that will replace this one. You should incorporate before merging.